ui/src/app/tasks-jobs/schedules/destroy/destroy.component.html

```diff
@@ -1,25 +1,17 @@
 <clr-modal [(clrModalOpen)]="isOpen" *ngIf="schedules" [clrModalClosable]="!isRunning">
-  <h3 *ngIf="schedules?.length === 1" class="modal-title">Confirm Delete Schedule</h3>
-  <h3 *ngIf="schedules?.length > 1" class="modal-title">Confirm Delete Schedules</h3>
+  <h3 *ngIf="schedules?.length === 1" class="modal-title">{{ 'schedules.destroy.title' | translate }}</h3>
+  <h3 *ngIf="schedules?.length > 1" class="modal-title">{{ 'schedules.destroy.title2' | translate }}</h3>
   <div class="modal-body" *ngIf="schedules.length === 1 && !isRunning">
-    This action will delete the schedule
-    <u
-      ><strong>{{ schedules[0].name }}</strong></u
-    >. Are you sure?
+    <div [innerHTML]="'schedules.destroy.description' | translate: {name: schedules[0].name}"></div>
   </div>
   <div class="modal-body" *ngIf="schedules?.length > 1 && !isRunning">
-    This action will delete the
-    <u
-      ><strong>{{ schedules.length }} schedule{{ schedules?.length > 1 ? 's' : '' }}</strong></u
-    >
-    listed below.<br />Are you sure?
-    <br />
+    <div [innerHTML]="'schedules.destroy.description2' | translate: {count: schedules.length}"></div>
     <table class="table table-striped">
       <thead>
         <tr>
-          <th class="left">Name</th>
-          <th class="left">Task name</th>
-          <th class="left">Platform</th>
+          <th class="left">{{ 'schedules.main.name' | translate }}</th>
```

Comment:
Texts like `Name` and `Platform` are shown on other pages.
Can we move these to `commons` translations ?